bvr_ref_compassion/model/invoice.py

```diff
@@ -42,15 +42,7 @@ def _check_bvr_ref(self, cr, uid, ids, context=None):
     _constraints = [(
         _check_bvr_ref,
         'Error: BVR ref should only contain number (max. 27) and spaces.',
-        ['bvr_reference'])]
-
-    def _get_bvr_ref(self, cr, uid, invoice, context=None):

```

Comment:
This code was never called (I can find no reference to this method, do you?). I don't know if it is because l10n_modules changed or not.